### PR TITLE
Improve user email experience.

### PR DESCRIPTION
**Pull request by rickfoxcroft**
Add functionality to send an html string as an email.

### CODE REVIEW

File: templates/template.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        }
+
+        .force-width-80 {
+            width: 80% !important;
+        }
+    </style>
+
+    <style type="text/css" media="screen">
+        @media screen {
+            /*Thanks Outlook 2013! http://goo.gl/XLxpyl*/
```
Comment (Hamo69) [inline templates/template.html]: /*Thanks Outlook 2013! http://goo.gl/XLxpyl*/

?

File: templates/template.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
```
Comment (Hamo69) [inline templates/template.html]: Should we make this Mech Rock branded?

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }, function (err, data) {
+        if (err) throw err
+        console.log('Email sent:');
+        console.log(data);
+    });
+
+    const response = {
+        statusCode: 200,
+        body: JSON.stringify({
+            message: 'Go Serverless v1.0! Your function executed successfully!',
```
Comment (Hamo69) [inline src/index.js]: Shall we customise this msg?

File: test/templateParserTest.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        'instance-id': "i-02eb595d1ea548b31",
+        state: "running"
+    }
+}
+
+var filename = '../templates/template.html';
+
+describe("Template Parser:", () => {
+
+    it('should open the html file', () => {
```
Comment (Hamo69) [inline test/templateParserTest.js]: I like the test. What other values can we obtain? I mean, to make it meaningful we need the account description / name and who created the instance

File: templates/template.html (hunk 1 above)
Comment (rickfoxcroft) [reply templates/template.html]: Will just remove.

File: test/templateParserTest.js (hunk 4 above)
Comment (rickfoxcroft) [reply test/templateParserTest.js]: Different Jira ticket.

File: templates/template.html (hunk 2 above)
Comment (Hamo69) [reply templates/template.html]: it means, should the template.html have the Mechanical Rock logo and the look and feel (ie. the colours / font)

**Comment by rickfoxcroft** [comment]
Struggling a bit, with automating the deployment of this. Help would be great.

**Pull request merged**